src/ggshield-webview/gitguardian-webview-view.ts

```diff
@@ -146,13 +146,56 @@ export class GitGuardianWebviewProvider implements vscode.WebviewViewProvider {
         </body>
         </html>
     `;
+    } else {
+      computedHtml = `
```

Comment:
In this case, the status bar will now be red and display "GitGuardian - Authentication Failed", wdyt ?

Reply:
Yes, that works for me, I think it's important that something visual change, so the user know that its authentication has fail. Otherwise it was a bit unclear if the extension was even doing something